src/download.tsx

```diff
@@ -138,6 +140,11 @@ class Download extends KalturaPlayer.core.BasePlugin {
     }
   }
 
+  _addBindings() {
+    this.eventManager.listen(this.downloadPluginManager, DownloadEvent.SHOW_OVERLAY, (e) => this.dispatchEvent(DownloadEvent.SHOW_OVERLAY, e.payload));
```

Comment:
because we need to dispatch the plugin events from the player `FakeEventTarget` (which wrapped inside the disaptchEvent method from the base plugin)